.travis.yml

```diff
@@ -37,6 +39,9 @@ before_install:
       export CXX="g++-4.7";
       export LINK="gcc-4.7";
       export LINKXX="g++-4.7";
+      export DOCKER_COPY="/tmp/alpine";
+      cp -r $TRAVIS_BUILD_DIR $DOCKER_COPY;
+      docker pull mhart/alpine-node:$TRAVIS_NODE_VERSION;
```

Comment:
There's not an official alpine image 🎉 
https://hub.docker.com/r/library/node/tags/6-alpine/